payments-core/src/main/java/com/stripe/android/model/PaymentMethod.kt

```diff
@@ -355,6 +355,7 @@ constructor(
             isVoucher = false,
             requiresMandate = false,
             hasDelayedSettlement = false,
+            afterRedirectAction = AfterRedirectAction.Poll(),
```

Comment:
We don't have tests for this.

Reply:
My bad, I originally just tested in the simulator with some logging.
Added unit tests to cover this.